tools/respec2html.js

```diff
@@ -43,7 +64,9 @@ cli
   .option("-w, --haltonwarn", "Abort if ReSpec generates warnings.", false)
   .option("--disable-sandbox", "Disable Chromium sandboxing if needed.", false)
   .option("--devtools", "Enable debugging and show Chrome's DevTools.", false)
-  .option("--verbose", "Log processing status to stdout.", false);
+  .option("--verbose", "Log processing status to stdout.", false)
+  .option("--localhost", "Spin up a local server to peform processing.", false)
```

Comment:
This message some a room for improvements. It looks a bit misleading right now.

Reply:
Feel free to change.... the current wording makes sense to me so I couldn't come up with anything better 😢.

Reply:
Can't think of any better either. Right now, it sounds like the server will perform processing..

Reply:
That's not entirely untrue... if you squint just right.